src/annotator/index.js

```diff
@@ -36,32 +36,38 @@ const appLinkEl = /** @type {Element} */ (
 
 function init() {
   const annotatorConfig = getConfig('annotator');
-  const isPDF = typeof window_.PDFViewerApplication !== 'undefined';
+  const sidebarConfig = getConfig('sidebar');
+  const { sidebarAppUrl } = sidebarConfig;
+
+  const frameConnector = new FrameConnector({
+    sidebarAppUrl,
+  });
```

Comment:
Yes, I noticed too. Well spotted. However, because only the `frameConnector.listen()` is called in the `host` frame the extra ports created are not been used in the `guest` frames.

However, I will address this more cleanly later.